spring-cloud-gcp-starters/spring-cloud-gcp-starter-config/src/main/java/org/springframework/cloud/gcp/config/GoogleConfigPropertySourceLocator.java

```diff
@@ -136,4 +144,9 @@ GoogleConfigEnvironment getRemoteEnvironment() throws IOException, HttpClientErr
 		}
 		return new MapPropertySource(PROPERTY_SOURCE_NAME, config);
 	}
+
+	@VisibleForTesting
```

Comment:
Do we really need this?
I'm pretty fine to have the getter for the property on the class.